### PR TITLE
[SaaS Connectors] Demonstrate Sentry Masking [#343]

### LINKED ISSUE #343: [SaaS Connector] Sentry Masking Test
# What

Sentry masking followup. Followup to #340.  Sentry config already has the update request spelled out, and testing has been completed against sentry's sandbox, but we don't have end-to-end tests in fidesops for that endpoint yet.

- Add end-to-end test of masking assignedTo field on project issues,
- Adjust access test to get data from project issues endpoint 

# Blockers
Our test sentry instance needs project issues data populated.

### PR DESCRIPTION
**Pull request by pattisdr**
# Purpose

Add an end-to-end test for sentry masking.  The only masking demonstrated is setting an issues "assignedTo" to None (which means an employee has submitted an erasure request, an edge case).  The only option to mask is to set it to an empty string - there's not a lot we can mask here.

# Changes
-  Adds an end-to-end test that assigns a user to an issue, and then masks it by setting it to an empty string.
- Also verifies `issues` returned in an access request.

# Checklist

- [ ] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md))
- [x] Good unit test/integration test coverage
- [ ] This PR contains a DB migration. If checked, the reviewer should confirm with the author that the [down_revision correctly references the previous migration](https://ethyca.github.io/fidesops/development/contributing_details/#alembic-migrations) before merging
- [x] The `Run Unsafe PR Checks` label has been applied, and checks have passed, if this PR touches any external services

# Ticket

Fixes #343

### CODE REVIEW

**Comment by pattisdr** [comment]
not sure what the issue is yet, this test is passing locally ^

File: tests/fixtures/saas/sentry_fixtures.py [hunk 1]
```diff
@@ -23,6 +23,11 @@ def sentry_secrets():
         "host": pydash.get(saas_config, "sentry.host") or os.environ.get("SENTRY_HOST"),
         "access_token": pydash.get(saas_config, "sentry.access_token")
         or os.environ.get("SENTRY_ACCESS_TOKEN"),
+        "erasure_access_token": pydash.get(saas_config, "sentry.erasure_access_token") or os.environ.get("SENTRY_ERASURE_TOKEN"),
+        "erasure_identity_email": pydash.get(saas_config, "sentry.erasure_identity_email") or os.environ.get("SENTRY_ERASURE_IDENTITY"),
```
[the remark this answers is not in the document]
Comment (pattisdr) [reply tests/fixtures/saas/sentry_fixtures.py:27]: I was mainly commenting here that we had to get credentials set up in a separate sentry account because we've been unable to login to the dashboard of the other.  We'd like to consolidate.  

I agree that the erasure identity should be separate from adaptors, we just need to consolidate into one sentry account.

[the remark this answers is not in the document]
Comment (galvana) [reply tests/fixtures/saas/sentry_fixtures.py:27]: The legacy adaptors repo had the concept of a `sar_identity` and an `rtf_identity` with the `rtf_identity` having some unique value appended to it (usually the unix timestamp). Maybe here we can move to the more user-friendly names of `access_identity` and `erasure_identity`. Not for this ticket but something to consider in the future as we standardize our testing.

[the remark this answers is not in the document]
Comment (pattisdr) [reply tests/fixtures/saas/sentry_fixtures.py:27]: good suggestion

**Review by galvana — APPROVED** [review]
Short and sweet, makes sense given the limited possibilities for erasure. Also, thanks for fixing the Makefile issue! This looks good to go.

**Comment by pattisdr** [comment]
thanks for the review @galvana
